docs/CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 Version 3.32.0-eisop1 (March ?, 2023)
-----------------------------------------
+
```

Comment:
Please undo all these unrelated changes to this file. Look at your other changes and undo unrelated changes.